scripts/superdesk-search/services/SearchService.js

```diff
@@ -448,6 +448,7 @@ export function SearchService($location, gettext, config, session) {
                     _.forEach(_.keys(item.es_highlight), function (key) {
                         item[key] = item.es_highlight[key][0];
                     });
+                    item.reRender = true; // used to make it re-render.
```

Comment:
imo would be better to create a new item here with `return angular.extend({}, item);`, rather then adding here a flag and create this new item later.